### PR TITLE
Revert "*: enlarge the default value of `max-merge-region-size`. (#8445)"

### PR DESCRIPTION
**Pull request by LykxSassinator**
This reverts commit aa85b6c0047b0f2af37c1bd271ee194af1c0dd56.

### What problem does this PR solve?

Issue Number: Ref https://github.com/pingcap/tidb/issues/55374

### What is changed and how does it work?

```commit-message
Revert the changes on the configuration of region-size. These changes will be delayed until v8.4.
```

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Code changes

- Has the configuration change
- Has HTTP API interfaces changed (Don't forget to [add the declarative for the new API](https://github.com/tikv/pd/blob/master/docs/development.md#updating-api-documentation))
- Has persistent data change

Side effects

- Possible performance regression
- Increased code complexity
- Breaking backward compatibility

Related changes

- PR to update [`pingcap/docs`](https://github.com/pingcap/docs)/[`pingcap/docs-cn`](https://github.com/pingcap/docs-cn):
- PR to update [`pingcap/tiup`](https://github.com/pingcap/tiup):
- Need to cherry-pick to the release branch

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by nolouch — APPROVED** [review]

**Review by lhy1024 — APPROVED** [review]

File: conf/config.toml [hunk 1]
```diff
@@ -111,9 +111,9 @@
 
 [schedule]
 ## Controls the size limit of Region Merge.
-# max-merge-region-size = 54
+# max-merge-region-size = 20
 ## Specifies the upper limit of the Region Merge key.
-# max-merge-region-keys = 540000
+# max-merge-region-keys = 200000
```
Comment (lhy1024) [inline conf/config.toml:116]: Do we also need to update doc in 8.3?

**Review by HuSharp — APPROVED** [review]

File: conf/config.toml (hunk 1 above)
Comment (LykxSassinator) [reply conf/config.toml:116]: All relative release notes have been reverted in doc and doc-cn.

**Review by niubell — APPROVED** [review]

**Pull request merged**